tests/transports-governance/test_governance_usage_tracking_hierarchical.py

```
@@ -0,0 +1,1001 @@
+        provider_stats_response = governance_client.get_usage_stats(provider="openai")
+        assert provider_stats_response.status_code in [
+            200,
+            400,
+        ]  # Might not be supported
+
+        # Test filtering by model (if supported)
+        model_stats_response = governance_client.get_usage_stats(model="gpt-4")
+        assert model_stats_response.status_code in [200, 400]  # Might not be supported
+
+        # Test date range filtering (if supported)
+        from datetime import datetime, timedelta
```

Ruff (0.12.2)

Redefinition of unused `timedelta` from line 20

Remove definition: `timedelta`

(F811)